### ISSUE TITLE
Recreated the model, but the accuracies are different from reported

### ISSUE THREAD

**Issue by anzr299** [CarDD-USTC/CarDD-USTC.github.io#8]
I have tried to replicate the DCN+ model and evaluated it on the test dataset. The mAP for small objects I am getting is 17.8 unlike the 34.6 reported in the paper.

I would like to discuss if I am doing something wrong here?

**Comment by ahmedxomar101** [comment]
@anzr299 Could you please share the performance on the other metrics? 

A screenshot would be optimal. Thanks.

**Comment by anzr299** [comment]
I had put it on an excel sheet since I deleted the checkpoints.
![image](https://github.com/CarDD-USTC/CarDD-USTC.github.io/assets/36276944/f38b686f-56de-4adb-834a-43418833cfde)

**Comment by ahmedxomar101** [comment]
@anzr299 Amazing. Thank you for sharing. 

Did you change the definitions for APs, m, l in your evaluation? It is a not nice trick to increase the performance of APs, m, l on paper

Check this part in the paper
![image](https://github.com/CarDD-USTC/CarDD-USTC.github.io/assets/44009418/656dcdfe-f8b1-48f7-83d5-c5f6417adfdc)

**Comment by anzr299** [comment]
Yeah, I did go through the paper and confirmed everything. I ran the same test script they gave. Also I would like to point out the part where the Learning rate in the code was 0.05 when it was mentioned in the paper to be 0.01.

**Comment by ahmedxomar101** [comment]
@anzr299 Very interesting! Would you mind if we talk privately? My LinkedIn account is listed on my profile.

**Comment by anzr299** [comment]
sure! I have sent a request.
